pom.xml

```diff
@@ -66,8 +66,7 @@
 
   <properties>
     <mavenVersion>3.0</mavenVersion>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.release>11</maven.compiler.release>
```

Comment:
It would make sense to  call JLink via toolchain, so there's no need for this requirement. Recently we've decided to make the minimum Java 8 for Maven plugins

Reply:
But jlink doesn't come with java 8?